flake.nix

```
@@ -0,0 +1,45 @@
+{
+    description = "Tool to simplify secure shell connections over AWS simple systems manager.";
```

We might as well use the correct current name of the AWS service:
```suggestion
    description = "Tool to simplify secure shell connections over AWS Systems Manager (SSM).";
```